app/uk/gov/hmrc/cdsimportsddsfrontend/domain/DocumentationType.scala

```diff
@@ -16,41 +16,57 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.domain
 
+case class WriteOffType(quantity: Option[String])
```

Comment:
is this used anywhere?

Reply:
I'll remove it.
It will be used in the future but not at the moment.